pkg/gardenlet/controller/backupentry/backup_entry_actuator.go

```diff
@@ -120,6 +120,9 @@ func (a *actuator) Delete(ctx context.Context) error {
 // by the current progress of the Flow execution.
 func (a *actuator) reportBackupEntryProgress(ctx context.Context, stats *flow.Stats) {
 	if err := kutil.TryUpdateStatus(ctx, kretry.DefaultRetry, a.gardenClient, a.backupEntry, func() error {
+		if a.backupEntry.Status.LastOperation == nil {
+			return fmt.Errorf("last operation of BackupEntry %s/%s is unset", a.backupEntry.Namespace, a.backupEntry.Name)
```

Comment:
Hm, why is this an error? Can't you simply initialize it to allow it being set in the next lines?

Reply:
Hmm, `ReportShootProgress` also does it like this:
https://github.com/gardener/gardener/blob/ed1dd896ddc16a64e47714c08531799d6ea93dde/pkg/operation/operation.go#L408-L410
I just did it analogously...
We could initialize `Status.LastOperation` here, but this is already done before starting the flow. I just added it here, because I hit nil pointers here caused by stale cache reads.